content/posts/windows/comment-coder-sous-windows/index.md

```diff
@@ -7,9 +7,7 @@ tags:
   - dev
 authors:
   - kud
-hero:
-  src: index.jpg
-  alt: Keyboard
+header:
```

Comment:
how do you force to have white title ? especially for this  header that has a black background.